Tasks/AzureKeyVault/package.json

```diff
@@ -0,0 +1,9 @@
+
+{
+  "name": "AzureKeyVault",
+  "main":"main.js",
+  "dependencies": {
+    "vsts-task-lib" : "^0.9.20",
```

Comment:
Tie to specific version. It will help you in controlling nested dependencies that might come along.